source/h5vol/H5VolReadWrite.h

```diff
@@ -33,10 +33,8 @@ extern herr_t H5VL_adios2_beginstep(const char *engine_name,
 
 extern herr_t H5VL_adios2_endstep(const char *engine_nane);
 
-static herr_t H5VL_adios2_introspect_opt_query(void *obj, H5VL_subclass_t cls,
-                                               int opt_type, hbool_t *supported)

```

Comment:
This old signature is actually the correct one, with the hbool_t * changed to a uint64_t * instead.